src/dbus_mpris.rs

```diff
@@ -562,12 +554,49 @@ async fn create_dbus_server(
         }
     });
 
+    let spotifyd_ctrls_interface: IfaceToken<()> = cr.register("io.github.spotifyd.Controls", |b| {
+        let local_spirc = spirc.clone();
+        b.method("VolumeUp", (), (), move |_, _, (): ()| {
+            local_spirc.volume_up();
+            Ok(())
+        });
+        let local_spirc = spirc.clone();
+        b.method("VolumeDown", (), (), move |_, _, (): ()| {
+            local_spirc.volume_down();
+            Ok(())
+        });
+
+        let mv_device_name = device_name.clone();
+        let sp_client = Arc::clone(&spotify_api_client);
+        b.method("TransferPlayback", (), ("status", ), move |_, _, (): ()| {
+            let device_id = get_device_id(&sp_client, &mv_device_name, false).flatten();
+            let status = if let Some(device_id) = device_id {
+                info!("Transferring playback to device {}", device_id);
+                match sp_client.transfer_playback(&device_id, Some(true)) {
+                    Ok(_) => "Playback transferred".to_string(),
+                    Err(err) => format!("Error {}", err)
+                }
+            } else {
+                let msg = format!("Could not find device with name {}", mv_device_name);
+                warn!("{}", msg);
+                msg
+            };
+            Ok((status, ))
```

Comment:
Instead of returning a string here, what do you think about reusing the built-in dbus error variants?
This would mean:
- return nothing (aka `Ok(())`) in case of success.
- return an error message wrapped in [`Err(dbus::MethodErr)`](https://docs.rs/dbus/latest/dbus/struct.MethodErr.html#method.failed) in case of failure

I guess that might be more dbus-like? My experience with interface design over there is limited, however, so I might be wrong.

Reply:
Done.